.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: go
 
 go:
+  - 1.6.x
+  - 1.7.x
```

Comment:
Let's remove 1.7 and 1.8 for now. Travis takes already long time to build.

Reply:
Alright, SGTM, I'll update that in a second.